pkg/volume/iscsi/iscsi.go

```diff
@@ -97,6 +97,11 @@ func (plugin *iscsiPlugin) newBuilderInternal(spec *volume.Spec, podUID types.UI
 	lun := strconv.Itoa(iscsi.Lun)
 	portal := portalBuilder(iscsi.TargetPortal)
 
+	iface := iscsi.ISCSIInterface
+	if iface == "" {
+		iface = "default"
```

Comment:
What I meant is that we can default it much earlier so that you know it's always populated and don't need to check it, and that users see the default.  But It's probably OK as is.